articles/users/search.md

```diff
@@ -292,7 +367,7 @@ When retrieving users in Auth0, there are three different API endpoints you can
 | Back of the office processes, such as changing a user's display name | [Users](#users) |
 | Searches involving user attributes | [Users](#users) |
 | Searches returning multiple users | [Users](#users) |
-| Operations requiring immediate consistency | [Users by Email](#users-by-email) |
-| Actions requiring user search as part of the Authentication Pipeline | [Users by Email](#users-by-email) |
+| Operations requiring immediate consistency | [Users by ID](#users-by-id) or [Users by Email](#users-by-email) |
+| Actions requiring user search as part of the Authentication Pipeline | [Users by ID](#users-by-id) or [Users by Email](#users-by-email) |
```

Comment:
Same comment on the `Authentication Pipeline`. Use a consistent term throughout the doc.